e2e/specs/menu_bar/view_menu.test.js

```diff
@@ -126,6 +138,28 @@ describe('menu/view', function desc() {
         zoomLevel.should.be.greaterThan(1);
     });
 
+    it('MM-44931 Zoom in when CmdOrCtrl+Shift+Plus is pressed', async () => {
```

Comment:
Thanks for adding these tests!
For better organization, I'd like to add this as part of the MM-T818 test case, which is in the same file. So what we can do here is put them both under a `describe()` with the label `MM-T818` and add this one as part 2. Have a look at the other test files and you can see how we split up the pieces.